ee_plugin/Map.py

```diff
@@ -37,6 +37,24 @@ def addLayer(image: ee.Image, visParams=None, name=None, shown=True, opacity=1.0
 
     ee_plugin.utils.add_or_update_ee_image_layer(image, name, shown, opacity)
 
+def getBounds():
```

Comment:
Does it make sense to make this compliant with the Code Editor? To be honest, I always use Map.getBounds(true) in the Code Editor, and almost never as a list or [W, S, E, N].

![image](https://user-images.githubusercontent.com/169821/69450248-067a0100-0d5d-11ea-9819-4366e61bc5bc.png)